UnityProject/Assets/Scripts/Core/Lifecycle/Despawn.cs

```diff
@@ -71,7 +72,15 @@ private static async Task<DespawnResult> Server(DespawnInfo info, bool skipInven
 			}
 		}
 
-		_ServerFireDespawnHooks(DespawnResult.Single(info));
+		try
+		{
+			_ServerFireDespawnHooks(DespawnResult.Single(info));
+		}
+		catch (Exception e)
```

Comment:
Is putting trycatches on these things a good idea? they reduce performance ever so slightly that they're a non-issue when it doesn't happen every now and then; but this is the despawn class, we despawn stuff all the time.

Reply:
Will it don't remov the object if it errors with out try catch